src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,42 @@
+import { PeopleTable } from "../components/PeopleTable";
+import * as peopleFromServer from '../api';
+import { useEffect, useState } from "react";
+import { Person } from "../types/Person";
+import { Loader } from "../components/Loader";
+
+
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loader, setLoader] = useState(false);
+
+
+  useEffect(() => {
+    setLoader(true)
+    peopleFromServer.getPeople().then(setPeople)
+      .catch(()=>setError(true))
+    .finally(()=>setLoader(false))
```

Comment:
Consider adding more detailed error handling to provide better feedback to the user. For example, you could log the error or display a specific error message based on the error type.